doc/en/user/source/community/webp/index.rst

```diff
@@ -0,0 +1,25 @@
+.. _webp_wms_output_format:
+
+WMS WebP output format
+============================
+
+This module adds the `WebP <https://developers.google.com/speed/webp>`_ WMS output format.
+
+WMS GetMap Request: *FORMAT=image/webp*
+
+Advantages of the WebP image format compared to other formats:
+
+* | WebP lossy images are about 30% smaller than comparable JPEG images.
+* | WebP supports transparency, typically providing 3x smaller file sizes compared to PNG.
+* | WebP supports :ref:`paletted images <tutorials_palettedimages>`, typically providing 20% smaller file sizes compared to PNG.
+
+WebP is supported by all modern browsers (`caniuse <https://caniuse.com/webp>`_).
+However, backwards compatibility can be built in on the client side.
+
+Because native libraries are used, not all platforms are supported.
```

Comment:
I was wondering if one had to install native libs, but it seems that for the supported architectures, the natives are included in the jar. Maybe this would clarify?

```suggestion
Because native libraries are used, not all platforms are supported. For those supported, no additional native library needs to be installed though.
```

Maybe a hint of what to do if the architecture is not supported could be useful... even if it's just "contributo to the gotson library development".... Just thinking out loud.